assets/js/filters/block-list-block.js

```diff
@@ -14,13 +14,7 @@ const { addFilter } = wp.hooks;
 const withDefaultAttributes = createHigherOrderComponent(
 	( BlockListBlock ) => {
 		class WrappedComponent extends Component {
-			constructor() {
-				super( ...arguments );
-
-				this.getAttributesWithDefaults = this.getAttributesWithDefaults.bind(
-					this
-				);
-			}
+			state = { mounted: false };
```

Comment:
If I understand the situation correctly, we don't need to re-render the component when `mounted` goes from `false` to `true`, right? If that's the case, could we store `mounted` as a class property instead of in the state? Not a big deal but that would prevent some unnecessary re-renders.

Reply:
hmm good point, let me give that a try.

Reply:
good call on that, switched to a class property and it works. Still need to flip it on `componentDidUpdate` though.